channels/tsmf/client/tsmf_main.c

```diff
@@ -34,6 +34,38 @@
 
 #include "tsmf_main.h"
 
+void tsmf_send_eos_response(IWTSVirtualChannelCallback* pChannelCallback, UINT32 message_id)
+{
+	wStream* s;
+	int status;
+	TSMF_CHANNEL_CALLBACK* callback = (TSMF_CHANNEL_CALLBACK*) pChannelCallback;
+
+	if (!callback)
+	{
+		DEBUG_TSMF("No callback reference - unable to send eos response!");
+		return;
+	}
+
+	if (callback->stream_id)
+	{
+		s = Stream_New(NULL, 24);
```

Comment:
Ok, just for reference - there are other areas in tsmf that do not check the return value for this call either. I will try to hit all cases with an updated pull request.